src/SldStyleParser.ts

```diff
@@ -196,20 +219,23 @@ class SldStyleParser implements StyleParser {
     const wellKnownName = _get(sldSymbolizer, 'Graphic[0].Mark[0].WellKnownName[0]');
     const externalGrahphic = _get(sldSymbolizer, 'Graphic[0].ExternalGraphic[0]');
     if (wellKnownName === 'circle') {
-      const strokeParams = _get(sldSymbolizer, 'Graphic[0].Mark[0].Stroke[0].CssParameter') || [];
-      const circleSymbolizer: CircleSymbolizer = {
-        kind: 'Circle',
-        opacity: _get(sldSymbolizer, 'Graphic[0].Opacity[0]'), // Could also come from fill-opacity
-        radius: _get(sldSymbolizer, 'Graphic[0].Size[0]'),
-        color: _get(sldSymbolizer, 'Graphic[0].Mark[0].Fill[0].CssParameter[0]._')
+      let circleSymbolizer: CircleSymbolizer = <CircleSymbolizer> {
+        kind: 'Circle'
       };
+      const strokeParams = _get(sldSymbolizer, 'Graphic[0].Mark[0].Stroke[0].CssParameter') || [];
+      const opacity = _get(sldSymbolizer, 'Graphic[0].Opacity[0]');
+      const radius = _get(sldSymbolizer, 'Graphic[0].Size[0]');
+      const color = _get(sldSymbolizer, 'Graphic[0].Mark[0].Fill[0].CssParameter[0]._');
+      if (opacity) { circleSymbolizer.opacity = opacity; }
```

Comment:
I personally would prefer line breaks for an if-statement even if it has just one statement in its body (also affects the next 2 lines).

Reply:
I allready wasn't sure about that. I'll switch it to multiline statements.